teckers/teckers/SignedInViewController.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  SignedInViewController.swift
+//  teckers
+//
+//  Created by Maggie Mendez on 7/24/19.
+//  Copyright © 2019 Teckers. All rights reserved.
+//
+
+import UIKit
+import Firebase
+import GoogleSignIn
+
+class SignedInViewController: UIViewController, GIDSignInUIDelegate {
+    @IBOutlet weak var signOutButton: UIButton!
+    @IBOutlet weak var emailLabel: UILabel!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setUpUI()
+    }
+    func setUpUI(){
```

Comment:
missing space

Reply:
I'll accommodate but Haydee proposed to use "SwiftLint" to format the code